### ISSUE TITLE
🐛[BUG] 最近3天录制直播经常断,导致原来2小时的直播,现在几分钟就重新开始一次,直播视频被切分成很多文件

### ISSUE THREAD

**Issue by 9ihbd2DZSMjtsf7vecXjz** [ihmily/DouyinLiveRecorder#851]
### ⚠️ 确认是否已存在类似问题

- [X] 我已经搜索过issues，没有找到类似问题

### 🔧 运行方式

使用源代码运行

### 🐍 如果是使用源代码运行，请选择你的Python环境版本

Python 3.12

### 💻 请选择你的系统环境

Windows 10

### ⚠️ 确认是否已经重试多次

- [X] 我已经尝试过多次，仍然出现问题

### 🕹 复现步骤

1.使用了1个月功能完全正常
2.最近3天开始直播录制几分钟-几十分钟就断开一次
不知道各位遇到了同样的问题吗? 怎么解决的,感谢分享

### 😯 问题描述

1.使用了1个月功能完全正常
2.最近3天开始直播录制几分钟-几十分钟就断开一次
不知道各位遇到了同样的问题吗? 怎么解决的,感谢分享

### 📜 错误信息

没有报错,直接直播终止,又重新开始

**Comment by d5f4sd6f84dfa** [comment]
有这个情况，就是时不时会断开录制好一会之后又能录了

**Comment by 9ihbd2DZSMjtsf7vecXjz** [comment]
> 有这个情况，就是时不时会断开录制好一会之后又能录了

是的 但是完整的一场直播会产生若干个碎片视频文件,毫无规律,比如一场直播2小时30分钟,以前可以直接录制完整的一个文件,现在可能分成5-10个几十分钟/十几分钟不等的视频,很乱. 其他软件不存在这个问题https://github.com/OriX6/liveRecordAuto

但是这个软件又不支持直播后执行脚本动作

**Comment by d5f4sd6f84dfa** [comment]
> > 有这个情况，就是时不时会断开录制好一会之后又能录了
> 
> 是的 但是完整的一场直播会产生若干个碎片视频文件,毫无规律,比如一场直播2小时30分钟,以前可以直接录制完整的一个文件,现在可能分成5-10个几十分钟/十几分钟不等的视频,很乱. 其他软件不存在这个问题https://github.com/OriX6/liveRecordAuto
> 
> 但是这个软件又不支持直播后执行脚本动作

 你这个网址的我用过，BUG好多

**Comment by jiqing112** [comment]
https://github.com/ihmily/DouyinLiveRecorder/issues/854#issuecomment-2564331201

你按照他的说，试试3.0.7版本呢

https://github.com/ihmily/DouyinLiveRecorder/releases/tag/v3.0.7

**Comment by 12414123123123** [comment]
> [#854 (comment)](https://github.com/ihmily/DouyinLiveRecorder/issues/854#issuecomment-2564331201)
> 
> 你按照他的说，试试3.0.7版本呢
> 
> https://github.com/ihmily/DouyinLiveRecorder/releases/tag/v3.0.7

我换了3.0.7稳定版后暂时没有出现跳帧的情况，但是有另一个bug，我在关闭分段开启ts转mp4的前提下，直播结束后没有自动将ts文件转码，这时ts文件也无法删除，提示文件已被ffmpeg占用
